marimo/_runtime/app_meta.py

```diff
@@ -11,16 +11,42 @@ class AppMeta:
     """
     Metadata about the app.
 
-    This is used to store metadata about the app
-    that is not part of the app's code or state.
+    This class provides access to runtime metadata about a marimo app, such as
+    its display theme and execution mode. Get an instance with `marimo.app_meta()`.
```

Comment:
nit: can we remove `Get an instance with `marimo.app_meta()`. and when used in `runtime.py`